modules/cluster-logging-collector-tuning.adoc

```diff
@@ -0,0 +1,178 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-collector.adoc
+
+[id="cluster-logging-collector-tuning_{context}"]
+= Advanced configuration for the log collector
+
+Cluster logging includes multiple Fluentd parameters that you can use for performance tuning. The parameters involve the Fluentd chunking behaviors.
+
+Fluentd collects log data into _chunks_. When Chunks are created, they are consider to be in the _stage_, where chunks get filled with data. When the chunk is full, Fluentd moves the chuck to the _queue_, where chunks are held before being flushed, or written out to their destination. Fluentd can fail to flush a chunk for a number of reasons, such as network issues or capacity issues at the destination. If a chunk cannot be flushed, Fluentd retries flushing as configured.
+
+By default in {product-title}, Fluentd uses the _exponential backoff_ retry method, where Fluentd doubles the time it waits between attempts to retry flushing, which helps reduce re-connect requests with the destination. You can disable exponential backoff and use the _periodic_ retry method instead, which retries writing off the chunks at a specified interval. 
+
+By default, Fluentd retries flushing the chunks indefinitely. You can change this behavior and discard chunks that cannot be properly flushed.
+
+You configure chunking and flushing behavior using the following parameters in the Cluster Logging Custom Resource (CR). The parameters are then automatically added to the Fluentd ConfigMap.
+
+[NOTE]
+====
+You can use these parameters to help determine the trade-offs between latency and throughput. If you want to optimize cluster logging for throughput, you could batch to reduce network packet count; configure larger buffers and queues, delay flushes, and set longer times between retries. To optimize for low latency, you could send data as soon as possible, avoid build-up of batches, have shorter queues and buffers, and use more frequent flush and retries.
```

Comment:
I suggest to make the consequence clear here that larger buffers need more space on the node filesystem